lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -339,3 +346,28 @@ def test_kerning():
     s = "AVAVAVAVAVAVAVAV€AAVV"
     fig.text(0, .25, s, size=5)
     fig.text(0, .75, s, size=20)
+
+
+def test_glyphs_subset():
+    fpath = str(_get_data_path("fonts/ttf/DejaVuSerif.ttf"))
+    chars = "these should be subsetted!"
+
+    # non-subsetted FT2Font
+    nosubfont = FT2Font(fpath)
+    nosubfont.set_text(chars)
+
+    # subsetted FT2Font
+    subfont = FT2Font(get_glyphs_subset(fpath, chars))
+    subfont.set_text(chars)
+
+    nosubcmap = nosubfont.get_charmap()
+    subcmap = subfont.get_charmap()
+
+    # all unique chars must be available in subsetted font
+    assert set(chars).issuperset(set([chr(key) for key in subcmap.keys()]))
```

Comment:
This looks to me like the comment is contradicting the code. `set(chars)` is the set of characters we want to be available, so it should be a subset (not superset) of the set of characters available in the reduced font.

Reply:
... but since the test passes, I wonder what's going on. Could you check what the sets on both sides of the `.issuperset` operation actually are? I wonder if there's a bug that makes the right-hand set be empty.

Reply:
Although the left and the right sides are valid (they print the set of characters):
```console
{'o', 'b', 'd', 's', ' ', '!', 'u', 'h', 'e', 't', 'l'}
```
It seems LHS is equal to RHS, which is why the tests pass. I agree it should be `.subset()`. (though I'm not able to recall why I put superset in the first place :/)